### ISSUE TITLE
Check again that performing experiments in parallel does not significantly impact the results

### ISSUE THREAD

**Issue by nelimee** [nelimee/sqt#15]
As this package is targeted towards 1-qubit state tomography, the idea of performing the different tomography experiments in parallel across the different qubits of a chip was raised. 

As far as I remember, this has been checked successfully at the beginning of the `sqt` implementation. But there is no trace/documentation of it, and this check happened a few years ago. 

In particular, it would be nice to have a complete script that could check that automatically. Here is how I see the process:
1. perform the parallel experiment, exactly what is done by default in `sqt` right now,
2. perform single-qubit experiment(s) (or multi-qubit experiments with non-adjacent qubits?),
3. post-process both experiments and check if the difference between the two results is within the sample-error or not.

There are two effects that might impact this experiment and prove the above assumption wrong:
1. cross-talk, i.e., error happening on nearby qubits when a quantum gate is applied. This type of error seems to be predominant when 2-qubit gates are used, but I did not see a lot of studies about potential cross-talk using single-qubit gates. [Simultaneous single-qubit driving of semiconductor spin qubits at the fault-tolerant threshold](https://www.nature.com/articles/s41467-023-39334-3) might be an interesting read for this.
2. correlated measurements, that can be seen as a special case of cross-talk for measurements. [Mitigating Coupling Map Constrained Correlated Measurement Errors on Quantum Devices](https://doi.org/10.1145/3581784.3607039) might be interesting here.
